examples/python/rgbd/main.py

```diff
@@ -23,6 +23,7 @@
 DEPTH_IMAGE_SCALING: Final = 1e4
 DATASET_DIR: Final = Path(os.path.dirname(__file__)) / "dataset"
 DATASET_URL_BASE: Final = "http://horatio.cs.nyu.edu/mit/silberman/nyu_depth_v2"
+DATASET_URL_BASE_ALTERNATE: Final = "https://static.rerun.io/rgbd_dataset"
```

Comment:
Let's make `static.rerun.io` the primary, and nyu.edu the fallback. That way we don't cost them bandwidth in the normal case.

Reply:
Makes sense